public/js/load_comment.js

```diff
@@ -0,0 +1,112 @@
+const addComment = (data)=>{
+  const comment = document.createElement('div');
+  comment.innerHTML= data;
+  comment.setAttribute("class","comment");
+  document.querySelector("body").appendChild(comment);
+  console.log("test");
+}
+
+  //management
+  document
+.querySelector('#load_more')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/soutenance14/ST/pullRequest?prid=8161867)
```suggestion
.querySelector("#load_more")
```